### PR TITLE
Career Page

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by asasmith** [comment]
designs
image

html
screenshot

Looks like these icons are larger than they should be


<details><summary>Browser metadata</summary>





```
Path:      /careers
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1288 x 810 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-40--airtec.netlify.app%2Fcareers&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1288x810&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-40--airtec.netlify.app/careers) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60cbea780e97ac0007616803&commentID=60d20040bc28c9c8580ffc81&resolution=resolved)

**Comment by kcbreuil** [comment]
@asasmith These SVGS have quite a lot of white space around them, I used transform to move the text up, let me know what you think about the spacing on this. Also resized the SVGS to be a bit smaller.

**Comment by asasmith** [comment]
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /careers
Browser:   Chrome 91.0.4472.106 on Mac OS 10.15.7
Viewport:  904 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-40--airtec.netlify.app%2Fcareers&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=904x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-40--airtec.netlify.app/careers) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60d355e4af88170007381538&commentID=60da7f5f684b691966d93b9b&resolution=resolved)

**Comment by asasmith** [comment]
What has changed here?
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /careers
Browser:   Chrome 91.0.4472.106 on Mac OS 10.15.7
Viewport:  1680 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-40--airtec.netlify.app%2Fcareers&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1680x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-40--airtec.netlify.app/careers) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60d355e4af88170007381538&commentID=60da7fac684b6917dbd939cc&resolution=resolved)

File: web/src/components/svgs/CompetitivePay.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export default function CompetitivePay({ text }) {
+    return (
+        <div>
+            <svg
+                id="prefix__Layer_1"
+                x={0}
+                y={0}
+                viewBox="0 0 72 72"
+                width="350"
+                height="350"
```
Comment (asasmith) [inline web/src/components/svgs/CompetitivePay.js:13]: I thought we talked about using a tailwind class on the svg to control

Comment (kcbreuil) [reply web/src/components/svgs/CompetitivePay.js:13]: Yes I had mentioned during our call that I had already implemented this logic so it was already in the PR. Will update now.

**Pull request merged**